userguide/advanceduse/reverse-tethering.rst

```diff
@@ -4,71 +4,60 @@ Reverse tethering
 This tutorial explains how you get your Ubuntu Touch device online using a USB cable and a computer with internet access.
 This is useful if there is no available Wi-Fi connection or you don't have a data subscription on your device.
 
-Steps
------
+Steps to Set Up Reverse Tethering
+---------------------------------
 
 #. Prepare your device (Ubuntu Touch) and your computer:
```

Comment:
*device* and *computer* should be in italics

Reply:
i kinda wonder if we should clarify that the commands apply to linux computers

Reply:
ah. yeah good point. maybe stick the word linux some where ...

Reply:
and done

Reply:
Oops looks like i had a poluted another pr, the linux part is now in too xd